test/unit/messaging/messaging.spec.ts

```diff
@@ -3534,42 +3470,56 @@ describe('Messaging', () => {
       },
     ];
 
+    const GAPIC_RESPONSE = [
```

Comment:
This was duplicated somewhere earlier in this file. Might want to make this a module-level constant so all test cases in this file can use it.

Reply:
Addressed!